pkg/service/teststats/teststats.go

```diff
@@ -17,26 +17,26 @@ import (
 type ProcStats struct {
 	logger                       lumber.Logger
 	httpClient                   http.Client
-	ExecutionResultInputChannel  chan core.ExecutionResult
+	ExecutionResultInputChannel  chan core.ExecutionResults
 	wg                           sync.WaitGroup
-	ExecutionResultOutputChannel chan core.ExecutionResult
+	ExecutionResultOutputChannel chan core.ExecutionResults
 }
 
 // New returns instance of ProcStats
 func New(cfg *config.NucleusConfig, logger lumber.Logger) (*ProcStats, error) {
 	return &ProcStats{
 		logger:                      logger,
-		ExecutionResultInputChannel: make(chan core.ExecutionResult),
+		ExecutionResultInputChannel: make(chan core.ExecutionResults),
 		httpClient: http.Client{
 			Timeout: global.DefaultHTTPTimeout,
 		},
-		ExecutionResultOutputChannel: make(chan core.ExecutionResult),
+		ExecutionResultOutputChannel: make(chan core.ExecutionResults),
 	}, nil
 
 }
 
 // CaptureTestStats combines the ps stats for each test
-func (s *ProcStats) CaptureTestStats(pid int32) error {
+func (s *ProcStats) CaptureTestStats(pid int32, CollectTestStats bool) error {
```

Comment:
nit: `collectStats` instead of `CollectTestStats`

Reply:
done